packages/cli/contracts/src/Tables.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+/* Autogenerated file. Do not edit manually. */
```

Comment:
does this need to regenerated now that the codegen paths have changed?

also how would you feel about this being a `codegen.sol` entry point? then all codegen things could live here
(this is assuming that we'll keep the "all identifiers must be unique" constraint)

Reply:
I very slightly prefer tables and types separate, but everything in codegen seems fine too.
No strong opinion on `tables` vs  `Tables` either, I'd probably go with `tables` in an empty repo, but in MUD for consistency `Tables` would fit better, unless we want to change some casings in `store`/`world` too

Reply:
> unless we want to change some casings in store/world too
do we have imports where the file is uppercase but doesn't export an uppercase thing with that same name?

my biggest beef with the uppercase `Tables` is I would expect it to export a `Tables` contract or something, but it doesn't, so lowercase is more clear

(this can all happen in a follow up too, doesn't need to block this PR)

Reply:
actually after looking into it not really, only `Types.sol` (for store it's not autogenerated), so we should then call it `types.sol` too

Reply:
now that we're calling these `enums` in the config, I wonder if the file should be `enums.sol` 

can still also re-export these from a `codegen.sol`